client/src/components/app.js

```diff
@@ -12,9 +12,10 @@ const App = Client({
 
 const CreativeCollab = () => (
   <div className="container" style={{ padding: 50 }}>
-    <h1>CreativeColab</h1>
+    <h1>CreativeColab1</h1>
```

Comment:
Typo?

Reply:
yep - wanted to see if things were changed in real time :)

Reply:
Yep they are! We have [HMR (hot module replacement)](https://webpack.js.org/guides/hot-module-replacement/) which will automatically refresh things as you edit them - pretty neat stuff